lang/uk/auth/login.php

```diff
@@ -1,8 +1,9 @@
 <?php
 
 return [
-    'title' => 'Вхід',
-    'email' => 'Електронна пошта',
+    'title_page' => 'Вхід',
+    'title' => 'Увійти',
+    'email' => 'Email або телефон',
```

Comment:
We will not do phone authentication
```suggestion
    'email' => 'Електронна пошта',
```